kie-server-parent/kie-server-remote/kie-server-client/src/main/java/org/kie/server/client/impl/UserTaskServicesClientImpl.java

```diff
@@ -870,6 +870,37 @@ public TaskInstance findTaskById(Long taskId) {
     }
 
     @Override
+    public List<TaskSummary> findTasksAssignedAsPotentialOwner(String userId, String filter, List<String> status, Integer page, Integer pageSize) {
```

Comment:
This is unnecessary duplication, it would be better to call the other findTasksAssignedAsPotentialOwner with default sorting.